commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -63,3 +64,8 @@ def get_edx_order_workflow_state_key(order: CTOrder) -> Optional[str]:
 
 def get_edx_is_sanctioned(order: CTOrder) -> bool:
     return get_edx_order_workflow_state_key(order) == TwoUKeys.SDN_SANCTIONED_ORDER_STATE
+
+
+def get_order_return_info_return_items(order: CTOrder) -> List[CTReturnItem]:
+    for item in map(lambda x: x.items, order.return_info):
+        return item
```

Comment:
You are returning after the first iteration only. I think we need to loop through all of the return info list to make our implementation logic future-proof, where orders can have multiple line items and/or multiple return info items

```suggestion
def get_order_return_info_return_items(order: CTOrder) -> List[CTReturnItem]:
    return_info_items = []
    for item in map(lambda x: x.items, order.return_info):
        return_info_items.extend(item)
        
    return return_info_items
```